include/oneapi/dpl/experimental/kt/esimd_radix_sort_cooperative.h

```diff
@@ -365,9 +365,16 @@ void cooperative(sycl::queue __q, _Range&& __rng, ::std::size_t __n) {
     }
     assert(__groups <= MAX_GROUPS);
 
-    auto p_sync = sycl::malloc_device<::std::uint32_t>(1024 + (__groups+2) * BIN_COUNT, __q);
+    const size_t full_buffer_size_part1 = sizeof(::std::uint32_t) * (1024 + (__groups + 2) * BIN_COUNT);
+    const size_t full_buffer_size_part2 = sizeof(KeyT) * (__groups * __group_block_size);
+    const size_t full_buffer_size = full_buffer_size_part1 + full_buffer_size_part2;
```

Comment:
Could you name the variables according to their purpose?

Reply:
fixed